app/experimenter/docs/adrs/0003-analysis-visualization-architecture-approach.md

```diff
@@ -0,0 +1,49 @@
+# Analysis Visualization: Architecture Approach
+
+* Status: accepted
+* Author: Marina Samuel
+* Deciders: Nimbus Team
+* Date: 2020-09-16
+
+## Context and Problem Statement
+
+Analysis and visualization of experiment result data needs to be [part of the experiment console](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md). The mockups based on user interviews are [here](https://www.figma.com/file/q6Xh9g6LrOj2OQ59LbAFXm/Experiment-Dashboard-Mockup).
+
+Some decisions need to be made about how the data will be pulled out of [Jetstream](https://github.com/mozilla/jetstream) and visualized within the console. This ADR will highlight how this will be done.
+
+
+## Decision Drivers
+
+Drivers for fetching data:
+* Data is easy to fetch and minimizes the amount of new state added
+
+Drivers for visualizing data:
+* Reuse existing [Graph Paper](https://github.com/graph-paper-org/graph-paper) UI components if possible
+* How easy or possible is it to integrate Svelte components into React (since Graph Paper is in Svelte)
+
+
+## Considered Options
+
+**Fetching Data**
+1. Add a POST endpoint to Jetstream to send data over to Experiment Console. This data would be saved in the Django database and fetched from the front-end when needed using GraphQL.
```

Comment:
Add a POST endpoint to ~~Jetstream~~Experimenter to send data over ~~to Experiment Console~~from Jetstream?

Reply:
oops, yes thats what I meant. Thanks